src/applications/disability-benefits/686c-674/config/helpers.js

```diff
@@ -158,6 +158,36 @@ export const isOutsideListLoopReturn = (
   };
 };
 
+export const hoursPerWeekUiSchema = {
+  'ui:title': 'Hours a week',
+  'ui:options': {
+    widgetClassNames: 'form-select-medium',
+  },
+  'ui:errorMessages': { required: 'Please enter a number' },
```

Comment:
As far as I know, error messages no longer include the word "Please".

Reply:
Thanks for the heads up. We currently use the phrase "Elease enter..." in 56 places across this form. Since this is not a new string and is just being moved I'm going to leave it here for now to maintain consistency and let our designer know that we might need to review all of the error messages.